packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -7,72 +7,65 @@
-      break;
-
-    case 'error':
-      borderColor = getColorV8('dangerHue', 700, props.theme);
-      backgroundColor = getColorV8('dangerHue', 600, props.theme);
-      foregroundColor = getColorV8('dangerHue', 100, props.theme);
-      anchorHoverColor = props.theme.palette.white;
-      anchorActiveColor = props.theme.palette.white;
-      focusColor = 'dangerHue';
-      break;
-
-    case 'warning':
-      borderColor = getColorV8('warningHue', 400, props.theme);
-      backgroundColor = getColorV8('warningHue', 300, props.theme);
-      foregroundColor = getColorV8('warningHue', 800, props.theme);
-      anchorHoverColor = getColorV8('warningHue', 900, props.theme);
-      anchorActiveColor = getColorV8('warningHue', 1000, props.theme);
-      focusColor = 'warningHue';
-      break;
-
-    case 'info':
-      borderColor = getColorV8('primaryHue', 300, props.theme);
-      backgroundColor = getColorV8('primaryHue', 200, props.theme);
-      foregroundColor = getColorV8('primaryHue', 700, props.theme);
-      anchorHoverColor = getColorV8('primaryHue', 800, props.theme);
-      anchorActiveColor = getColorV8('primaryHue', 900, props.theme);
-      focusColor = 'primaryHue';
-      break;
+  let focusVariable;
+
+  if (['error', 'success'].includes($alertType)) {
+    const borderVariable =
+      $alertType === 'success' ? 'border.successEmphasis' : 'border.dangerEmphasis';
+    const backgroundVariable =
+      $alertType === 'success' ? 'background.successEmphasis' : 'background.dangerEmphasis';
+    const foregroundVariable =
+      $alertType === 'success' ? 'foreground.success' : 'foreground.danger';
+
+    borderColor = getColor({ variable: borderVariable, light: { offset: 100 }, theme });
+    backgroundColor = getColor({ variable: backgroundVariable, theme });
+    foregroundColor = getColor({ variable: foregroundVariable, light: { offset: -600 }, theme });
+    focusVariable =
+      $alertType === 'success' ? 'foreground.successEmphasis' : 'foreground.dangerEmphasis';
+    anchorHoverColor = theme.palette.white;
+    anchorActiveColor = theme.palette.white;
+  } else {
+    const borderVariable =
+      $alertType === 'warning' ? 'border.warningEmphasis' : 'border.primaryEmphasis';
+    const backgroundVariable =
+      $alertType === 'warning' ? 'background.warningEmphasis' : 'background.primaryEmphasis';
+    const foregroundVariable =
+      $alertType === 'warning' ? 'foreground.warning' : 'foreground.primary';
+
+    borderColor = getColor({ variable: borderVariable, light: { offset: -300 }, theme });
+    backgroundColor = getColor({ variable: backgroundVariable, light: { offset: -400 }, theme });
+    foregroundColor = getColor({ variable: foregroundVariable, light: { offset: 100 }, theme });
+    anchorHoverColor = getColor({ variable: foregroundVariable, light: { offset: 200 }, theme });
+    anchorActiveColor = getColor({ variable: foregroundVariable, light: { offset: 300 }, theme });
```

Comment:
I think it would be better to use the `offset: xx` parameter throughout rather than `light: { offset: xx }` as the intent is to adjust theme-agnostic (which we know is now always forced to `light`).

Reply:
Updated